base_classes/NXdata.nxdl.xml

```diff
@@ -180,29 +174,35 @@
 		      mr: float[100]  --> the default independent data
 		  
 		  The field named in the ``signal`` attribute **must** exist, either
-		  directly as a dataset or defined through a link.
+		  directly as a field or defined through a link.
 		
-		* The group ``axes`` attribute will name the
-		  *dimension scale* associated with the plottable data.
+		* If available, the standard deviations of the data are to be
+		  stored in a field of the same rank and dimensions, either with the 
+		  name ``DATA_errors`` or with a name specified in the data's 
+		  ``uncertainties`` attribute. 
 
-		If available, the standard deviations of the data are to be
-		stored in a data set of the same rank and dimensions, with the name ``errors``. 
-
-		* For each data dimension, there should be a one-dimensional array
-		  of the same length.
-		* These one-dimensional arrays are the *dimension scales* of the
-		  data,  *i.e*. the values of the independent variables at which the data
-		  is measured, such as scattering angle or energy transfer.
+		* For each data dimension of size ``n``, there should be a 
+		  one-dimensional array of size ``n`` or ``n+1``, which contain the 
+		  *dimension scales* of the data,  *i.e*., the values of the independent 
+		  variables at which the data is measured, such as scattering angle or 
+		  energy transfer (size ``n``) or the bin boundaries when the data is
+		  histogrammed (size ``n+1 ``).
```

Comment:
Is n+1 widely supported? I think we only ever mandated the same length.

Reply:
That has been the standard way time-of-flight arrays have been stored since the beginning of NeXus. I just checked that both ARCS and MERLIN files store the time-of-flights as bin boundaries, so it's still in common usage. There is a 'histogram_offset' attribute that could be used in conjunction with bin centers, but I have documentation from 2002, which states that it is not mandatory.